apps/avifdec.c

```diff
@@ -36,6 +36,9 @@ static void syntax(void)
     printf("    -u,--upsampling U : Chroma upsampling (for 420/422). automatic (default), fastest, best, nearest, or bilinear\n");
     printf("    -r,--raw-color    : Output raw RGB values instead of multiplying by alpha when saving to opaque formats\n");
     printf("                        (JPEG only; not applicable to y4m)\n");
+    printf("    --index           : When decoding an image sequence or progressive image, specify which frame index to decode (Default: 0)\n");
```

Comment:
For a progressive image, it seems that the default should be the best layer, which has the largest frame index. Is it easy to implement that?

Reply:
If you don't pass `--progressive` at all, you will get the non-progressive behavior from the progressive AVIF, which will be the topmost/best layer.

Reply:
Thanks for the explanation. Perhaps in line 40 we can change

> If a progressive image is encountered,
to

> If a progressive image is encountered and --progressive is passed,